### ISSUE TITLE
Segmentation fault after upgrading to v2.13.x

### ISSUE THREAD

**Issue by tsironis** [FelixKratz/SketchyBar#312]
## Issue description

Today I noticed an issue when I clicked on an empty part of the sketchybar. I'm certain that this didn't happen yesterday, I guess it started after I upgrade to v2.13.0.

Initially, I thought that there is a bug in my config that I didn't notice but after commenting out almost everything in my sketchybarrc file I got exactly the same behaviour. Upgrading to v2.13.1 earlier today did not solve the problem. 

I don't have any experience debugging or exporting stacktrace for C process, so please advise on how to provide more info about these processes.

## Steps to reproduce the issue

1. brew services stop sketchybar 
2. Use sketchybar v2.13.x with a minimal config - no items added
3. Run sketchybar via a terminal
4. Click anywhere on the bar
5. The sketchybar process on the terminal window should crash with a segmentation fault

## What's the expected result?

Clicking on the actual bar and not a bar item that uses a `click_script` should not produce a segmentation fault or restart the sketchybar process.

#### What's the actual result?

https://user-images.githubusercontent.com/782448/211284851-a5baca02-7907-461e-bff1-0323c9b1d85d.mov

## Sketchybar config

```
#!/usr/bin/env bash

# ITEMS="$HOME/.config/sketchybar/items"
# PLUGINS="$HOME/.config/sketchybar/plugins"
FONT="Liga SFMono Nerd Font"
#
source "$HOME/.config/sketchybar/colors.sh" # Loads all defined colors

# Setting up and starting the helper process
# HELPER=git.felix.helper
# killall helper
# cd $HOME/.config/sketchybar/helper && make
# $HOME/.config/sketchybar/helper/helper $HELPER > /dev/null 2>&1 &

sketchybar -m --bar blur_radius=50      \
    color=$BAR_COLOR                    \
    height=32                           \
    --default background.padding_left=8 \
    icon.font="$FONT:Bold:14.0"         \
    label.font="$FONT:Semibold:13.0"    \
    background.padding_right=5          \
    background.corner_radius=8          \
    icon.padding_right=5

# Left items
# source "$ITEMS/apple.sh"
# source "$ITEMS/spaces.sh"
# source "$ITEMS/front_app.sh"
#
# # Right items
# source "$ITEMS/weather.sh"
# source "$ITEMS/calendar.sh"
# source "$ITEMS/input.sh"
# source "$ITEMS/mic.sh"
# source "$ITEMS/sound.sh"
# source "$ITEMS/wifi.sh"
# source "$ITEMS/battery.sh"
# source "$ITEMS/divider.sh"
# source "$ITEMS/cpu.sh"
# # source "$ITEMS/login_animation.sh"
#
# # Forcing all item scripts to run (never do this outside of sketchybarrc)
# sketchybar --update
#
# echo "✨ sketchybar configuration loaded.."
#
```

**Comment by FelixKratz** [comment]
Released a hotfix release! Should be fine with v2.13.2

**Comment by tsironis** [comment]
Worded like a charm! Thank you.
